r_code/synthetic_dgp.R

```diff
@@ -630,7 +620,10 @@ NoisifyDraw <- function(data_inp, seed, log_output = T, sig_y = 0.2) {
 #'
 #' @return Tibble with all individual level work completed.
 .UnitLevelSimulation <- function(n_inp,
-                                 type_inp,
+                                 type_inp = c(
+                                   "random", "observables",
+                                   "unobservables"
+                                 ),
```

Comment:
If you want to restrict to these, I think a corresponding `match.arg` is necessary.

Reply:
This was the case where it is only a parameter to be passed to another function -- do we still need the match.arg?